modelon/impact/client/entities/case.py

```diff
@@ -127,6 +129,66 @@ def simulation_log_level(self, simulation_log_level: str):
         self._analysis['simulation_log_level'] = simulation_log_level
 
 
+class CustomArtifact:
+    """CustomArtifact class"""
+
+    def __init__(
+        self,
+        workspace_id: str,
+        experiment_id: str,
+        case_id: str,
+        data: Dict[str, str],
+        exp_sal,
+    ):
```

Comment:
Is there any reason to have `Dict` as argument instead of explicit:
```
 def __init__(
        self,
        workspace_id: str,
        experiment_id: str,
        case_id: str,
        exp_sal,
        artifact_id: str,
        download_as: Optional[str] = None
    ):
```

Reply:
@iakovn  I was thinking that if in the future we a add more keys to the response to the GET /custom-artifacts endpoint, it probably might make more sense to keep it as a dict here.